tidb-cloud/changefeed-sink-to-mysql.md

```diff
@@ -115,21 +117,6 @@ After completing the prerequisites, you can sink your data to MySQL.
 SET GLOBAL tidb_gc_life_time = '10m';
 ```
 
-## Delete a Sink
-
-1. Navigate to the **Changefeed** tab of a cluster.
-2. Click the corresponding **Changefeed** card, and click **Delete**.
-
-## Pause or resume a Sink
-
-1. Navigate to the **Changefeed** tab of a cluster.
-2. Click the corresponding **Changefeed** card, and click **Pause** or **Resume**.
-
-## Query TiCDC RCUs
-
-1. Navigate to the **Changefeed** tab of a cluster.
-2. You can see the current TiCDC Replication Capacity Units (RCUs) in the upper-right corner of the page.
-
 ## Restrictions
 
 Because TiDB Cloud uses TiCDC to establish changefeeds, it has the same [restrictions as TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview#restrictions).
```

Comment:
```suggestion
- For each TiDB Cloud cluster, you can create up to 10 MySQL changefeeds.
- Because TiDB Cloud uses TiCDC to establish changefeeds, it has the same [restrictions as TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview#restrictions).
```
@IANTHEREAL @ran-huang PTAL